modules/op-configuring-pipelines-as-code-logging.adoc

```diff
@@ -0,0 +1,151 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-pipelines-as-code-logging_{context}"]
+= Configuring {pac} logging 
+
+You can configure {pac} logging by editing the `pac-config-logging` config map in the `TektonConfig` custom resource (CR).
+
+.Prerequisites
+
+* {pac} is installed on your cluster.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Administration* → *CustomResourceDefinitions*.
+
+. Use the *Search by name* box to search for the `tektonconfigs.operator.tekton.dev` custom resource definition (CRD) and click *TektonConfig* to see the CRD details page.
```

Comment:
```suggestion
. Use the *Search by name* box to search for the `tektonconfigs.operator.tekton.dev` custom resource definition (CRD) and click *TektonConfig* to see the CRD *Details* page.
```
I think. I don't have a cluster up right now to see the UI, but the name of the page should be bolded.

Reply:
I'll check and make the appropriate change, thank you!

Reply:
i just checked, it should be *Details*, thank you for noticing!